### PR TITLE
[Term Entry] Python:NumPy built-in-functions .median()

### LINKED ISSUE #4472: [Term Entry] Python:NumPy built-in-functions Median
### Reason/inspiration (optional)

"We would like a new term entry on the `built-in-functions` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): Median. The entry should go in a new file under `docs/content/numpy/concepts/built-in-functions/terms/median/median.md`.

The entry should describe the term and how it is used within statistical analysis. Also include: 
- An "Example" section that explains readers how to use the median statistical function.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by HalfEatenApple**
### Description

- Added the numpy.median() term for NumPy
- Closes #4472 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+```shell  
+2.5
+[1.5 2.5 3.5]
+[1. 4.]
+```
+
+## Codebyte Example
+
+The following example creates a 2-dimensional array composed of random integers (between 1 and 20). Then uses `.median()` to calculate the median, with various values of the `axis` parameter.
```
Comment (mamtawardhani) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:59]:
```suggestion
The following example creates a 2-dimensional array of random integers (between 1 and 20) and calculates the median using `.median()`, demonstrating different uses of the `axis` parameter.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
@HalfEatenApple The entry was nicely written!

However, there were some minor changes required, I have made some suggestions and comments wherever the changes are required, please make those changes at your earliest convenience.

**Review by mamtawardhani — APPROVED** [review]

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+```
+
+The `a` parameter is required and represents the array of elements to calculate the median from. All other parameters are optional.
+
+Parameters of the `.median()` function:
+
+- `a`: (Input) The array of elements to calculate the median from.
+- `axis`: (Default = None) An int or tuple of ints specifying the axis/axes along which to calculate the median. By default, it considers the array to be flattened (works on all axes). `axis = 0` works along the column, and `axis = 1` works along the row.
+- `out`: (Default = None) A different array to place the result. It must have the same shape as the expected result.
+- `keepdims`: (Default = False) A boolean; if `True` will keep reduced axes in the result as dimensions with size one.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:32]:
````suggestion
```pseudo
numpy.median(a, axis=None, out=None, overwrite_input=False, keepdims=False)
```

- `a`: The array of elements to calculate the median from.
- `axis`: The axis or axes along which to calculate the median. By default, it considers the array to be flattened (works on all axes). `axis=0` works along the column and `axis=1` works along the row.
- `out`: A different array to place the result. It must have the same shape as the expected result.
- `overwrite_input`: A parameter which, if `True`, allows memory usage of the array for calculations.
- `keepdims`: A parameter which, if `True`, keeps reduced axes in the result as dimensions with size one.

> **Note:** The `a` parameter is the only required parameter for this function. All other parameters are optional.
````

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## Example
+
+The following example creates an array then applies a few `.median()` operations, and returns each result to the console.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:36]:
```suggestion
The following example creates an array, then applies a few `.median()` operations, and returns each result to the console:
```

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+a = np.array([[0,1,2],[3,4,5]])
+
+print(np.median(a))
+#computes the median of the entire array
+
+print(np.median(a, axis=0))
+#computes the median along the vertical axis (column) of the array
+
+print(np.median(a, axis=1))
+#computes the median along the horizontal axis (row) of the array
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:50]:
```suggestion
# Computes the median of the entire array

print(np.median(a, axis=0))
# Computes the median along the vertical axis (column) of the array

print(np.median(a, axis=1))
# Computes the median along the horizontal axis (row) of the array
```

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+```shell  
+2.5
+[1.5 2.5 3.5]
+[1. 4.]
+```
+
+## Codebyte Example
+
+The following example creates a 2-dimensional array of random integers (between 1 and 20) and calculates the median using `.median()`, demonstrating different uses of the `axis` parameter.
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:63]:
```suggestion
The following example creates a 2-dimensional array of random integers (between 1 and 20) and calculates the median using `.median()`, demonstrating different uses of the `axis` parameter:
```

File: content/numpy/concepts/built-in-functions/terms/median/median.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+print("The median along the flattened array is:")
+print(np.median(randarr))
+
+print("The median along the vertical axis (axis = 0) is:")
+print(np.median(randarr, axis=0))
+
+print("The median along the horizontal axis (axis = 1) is:")
+print(np.median(randarr, axis=1))
+
+print("The median along the horizontal axis (axis = 1) with keepdims = True is:")
```
Comment (Sriparno08) [inline content/numpy/concepts/built-in-functions/terms/median/median.md:81]:
```suggestion
print("The median along the vertical axis (axis=0) is:")
print(np.median(randarr, axis=0))

print("The median along the horizontal axis (axis=1) is:")
print(np.median(randarr, axis=1))

print("The median along the horizontal axis (axis=1) with 'keepdims=True' is:")
```

**Review by Sriparno08 — COMMENTED** [review]
Hey @HalfEatenApple, I've listed all the modifications from my side that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**